protected/components/views/patientDiagnosesAndMedicationsWidget.php

```diff
@@ -24,7 +24,7 @@
                             <?php foreach ($this->patient->secondarydiagnoses as $diagnosis): ?>
                                 <tr>
                                     <td><?php echo $diagnosis->disorder->fully_specified_name; ?></td>
-                                    <td><?php echo ($diagnosis->is_confirmed !== 0) ? 'Confirmed' : 'Unconfirmed'; ?></td>
+                                    <td><?php echo ($diagnosis->is_confirmed != 0) ? 'Confirmed' : 'Unconfirmed'; ?></td>
```

Comment:
Was there any reason that the logic used in the patient summary screen can't apply here? ($diagnosis->is_confirmed === 0 ? 'Unconfirmed' : 'Confirmed')